Source/WebKit/UIProcess/API/glib/WebKitError.h.in

```diff
@@ -36,7 +36,10 @@ G_BEGIN_DECLS
 #define WEBKIT_JAVASCRIPT_ERROR                  webkit_javascript_error_quark ()
 #define WEBKIT_SNAPSHOT_ERROR                    webkit_snapshot_error_quark ()
 #define WEBKIT_USER_CONTENT_FILTER_ERROR         webkit_user_content_filter_error_quark ()
-#define WEBKIT_WEB_EXTENSION_MATCH_PATTERN_ERROR webkit_web_extension_error_quark ()
+#if ENABLE(2022_GLIB_API)
+#define WEBKIT_WEB_EXTENSION_ERROR               webkit_web_extension_error_quark ()
+#endif
+#define WEBKIT_WEB_EXTENSION_MATCH_PATTERN_ERROR webkit_web_extension_match_pattern_error_quark ()
```

Comment:
I read this as `webkit_web_extension_error_quark()` wasn't previously defined, so `WEBKIT_WEB_EXTENSION_MATCH_PATTERN_ERROR` has been broken?

Maybe worth documenting in the commit at least.